### PR TITLE
AP_Scripting: Add baro health checks

### PR DESCRIPTION
**Pull request by haydendonald**
Add health check bindings for the barometer

### CODE REVIEW

File: libraries/AP_Scripting/generator/description/bindings.desc [hunk 1]
```diff
@@ -334,6 +334,8 @@ singleton AP_Baro method get_pressure float
 singleton AP_Baro method get_temperature float
 singleton AP_Baro method get_external_temperature float
 singleton AP_Baro method get_altitude float
+singleton AP_Baro method all_healthy boolean
```
Comment (IamPete1) [inline libraries/AP_Scripting/generator/description/bindings.desc:337]: I don't think we need all healthy.

File: libraries/AP_Scripting/generator/description/bindings.desc [hunk 2]
```diff
@@ -334,6 +334,8 @@ singleton AP_Baro method get_pressure float
...
+singleton AP_Baro method healthy boolean uint8_t 0 BARO_MAX_INSTANCES
```
Comment (IamPete1) [inline libraries/AP_Scripting/generator/description/bindings.desc:338]: We get primary for all the others. Its a bit odd being able to get the health per instance but not anything else and no way to tell which is primary. So there is no way to match up the heath reading to the other values.

Comment (haydendonald) [reply libraries/AP_Scripting/generator/description/bindings.desc:338]: Sorry i should've specified the reason. We're looking into testing using LUA thus why we are wanting to check the individual health of each sensor.

**Comment by haydendonald** [comment]
Closing i'll combine PR's #23622, #23621 and #23619 into one.

**Pull request closed without merging**